test/libsolidity/syntaxTests/types/empty_tuple_function_self.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.4.3;
+contract C {
+    function a() public {
+        (a(), 7);
```

Comment:
I don't think this test case adds to coverage.

Reply:
Agree :)